com.unity.ml-agents/Runtime/Sensors/StackingSensor.cs

```diff
@@ -6,6 +6,8 @@ namespace MLAgents.Sensors
     /// For example, 4 stacked sets of observations would be output like
     ///   |  t = now - 3  |  t = now -3  |  t = now - 2  |  t = now  |
     /// Internally, a circular buffer of arrays is used. The m_CurrentIndex represents the most recent observation.
+    ///
+    /// Currently, compressed and multidimensional observations are not supported.
     /// </summary>
     internal class StackingSensor : ISensor
```

Comment:
was this intended to be public?

Reply:
ugh, forgot to do the main point of the PR.

Reply:
Fixed. Took me a while to remove the palm from my face.